CONTRIBUTING.md

```diff
@@ -28,6 +28,8 @@ See [our issue template](https://github.com/sinonjs/sinon/blob/master/.github/)
 
 ## Contributing to the code base
 
+Read [GitHub's Project Forking Guide](https://guides.github.com/activities/forking/) to learn about the general contribution process. 
```

Comment:
Could you just place it as the second paragraph, below the current one? It's more of a detail, and now it stands out as the main intro.

Possibly it could be phrased as "Make sure you have read GitHub's guide on forking. It explains the general contribution process and key concepts."